pom.xml

```diff
@@ -31,6 +31,7 @@
         <hazelcast.version>4.2</hazelcast.version>
 
         <junit.version>4.13.2</junit.version>
+        <apache.commons.version>3.12.0</apache.commons.version>
```

Comment:
We keep all Hazelcast modules dependency-free. Could you please remove the apache commons dependency?

Reply:
Thanks. Dependency removed.